### DISCUSSION TITLE
Transferring a domain

### DISCUSSION THREAD

**Discussion by kaisiant** [vercel/community#2712] [Help]
### Summary

I am trying to transfer a domain from namecheap. It keep mentioned that the domain already exists in my account but I cant find it.

<img width="933" alt="image" src="https://github.com/vercel/community/assets/26655111/94f29060-a881-44fc-82e6-695c564fe927">
<img width="1425" alt="image" src="https://github.com/vercel/community/assets/26655111/14191d92-baa2-42af-bc2a-fad076e80dbc">

### Steps to Reproduce

1. Click Transfer In
2. Input domain and auth code
3. Click Transfer In
4. The domain xxxxx already exists.

**Comment by amyegan** [comment]
Hi @kaisiant. It looks like that domain is associated with a different account. Do you have a second account where this domain may have been used?

**Reply by kaisiant** [reply]
No. Is it possible to help me find out the domain associated with which account?

**Reply by kaisiant** [reply]
I found the root cause. One of my colleagues added this domain before unlocked to his account so I cannot add this domain now. I think this is a bug, someone can steal the domain and block the owner of the domain to register the custom domain to his vercel account.
